sources/europe/at/TirisOrthofotoCIR.geojson

```diff
@@ -0,0 +1,954 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "tirol.gv.at-orthofoto",
+        "attribution": {
+            "text": "tiris.tirol.gv.at",
+            "required": false
+        },
+        "name": "Tiris: orthophoto CIR",
```

Comment:
I'm not sure everyone knows what *CIR* stands for. Maybe just call it "infrared" or so would be better?